packages/fast-components-react-base/src/slider/slider.tsx

```diff
@@ -295,6 +299,7 @@ class Slider extends Foundation<SliderHandledProps, SliderUnhandledProps, Slider
                 slider,
                 [slider__disabled, this.props.disabled],
                 [slider__dragging, this.state.isDragging],
+                [slider__touchDragging, this.state.isTouchDragging],
```

Comment:
Actually, should there be another class and callback for this? Because I would actually assume touch dragging is the same as mouse dragging, both visually and for the information passed as a callback. This way you can just check to see if there is dragging.

Reply:
Developers may want to have different visuals or transitions when in touch dragging vs mouse dragging, so I think there are valid use cases for keeping track of both separately.

Reply:
are both `slider__dragging` and `slider__touchDragging` applied when a user is touch dragging, or are dragging and touchDragging mutually exclusive?

Reply:
made changes so we always apply the dragging class while dragging for mouse or touch, then apply a touch dragging class if we are doing that.  renamed internal state props to be more clear as they are mutually exclusive.

Reply:
Looks great - thanks